LICENSE-CONTENT

```diff
@@ -0,0 +1,427 @@
+Attribution-ShareAlike 4.0 International
```

Comment:
I would like something here to explain more explicitly what this applies too. (Even if it redirects you to the documentation.)